pkg/db/db.go

```diff
@@ -83,7 +85,50 @@ func (o Organization) KeysAsSet() (*jwk.Set, error) {
 	for _, key := range o.Keys {
 		maps = append(maps, key.(map[string]interface{}))
 	}
-	return crypto.MapsToJwkSet(maps)
+	result, err := crypto.MapsToJwkSet(maps)
+	if err != nil {
+		return nil, err
+	}
+	// Support deprecated PublicKey
+	if o.PublicKey != nil {
+		key, err := crypto.PemToPublicKey([]byte(*o.PublicKey))
+		if err != nil {
+			return nil, err
+		}
+		pubKey, err := jwk.New(key)
+		if err != nil {
+			return nil, err
+		}
+		result.Keys = append(result.Keys, pubKey)
+	}
+	return result, nil
+}
+
+// HasKey checks whether the given key is owned by the organization
+func (o Organization) HasKey(key jwk.Key, instant time.Time) (bool, error) {
```

Comment:
Why the word instance?

Reply:
"instant" is actually a term from the Java 8 date API. I used it since it's discouraged to use package names ("time") as variable names. I'll rename it to "validAtMoment".